key/keyutil/keyutil.go

```diff
@@ -0,0 +1,51 @@
+package keyutil
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"strconv"
+
+	"github.com/plprobelab/go-kademlia/key"
+)
+
+var rng = rand.New(rand.NewSource(299792458))
+
+// Random returns a KadKey of length l populated with random data.
+func Random(l int) key.KadKey {
+	buf := make([]byte, l)
+	rng.Read(buf)
+	return buf
+}
+
+// RandomWithPrefix returns a KadKey of length l having a prefix equal to the bit pattern held in s.
+func RandomWithPrefix(s string, l int) key.KadKey {
```

Comment:
- the comment should mention the maximum prefix of 64 bits.
- since there are input checks there could also be a check that `len(s) <= l*8` is true

Reply:
done